### PR TITLE
[simbody] new port

### LINKED ISSUE #2955: Add simbody
It would be great to have a port of the Simbody library ( https://github.com/simbody/simbody ).

Debian package : https://packages.debian.org/unstable/source/simbody .
Homebrew formula (not from core repository) : https://github.com/osrf/homebrew-simulation/blob/master/simbody.rb .
Arch package: https://aur.archlinux.org/packages/simbody/ . 
Repology page: https://repology.org/metapackage/simbody/versions .

### PR DESCRIPTION
**Pull request by Ace314159**
This adds the simbody port

- #### What does your PR fix?  
  Fixes #2955 and #7705

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by Ace314159** [comment]
@NancyLi1013 I have made https://github.com/simbody/simbody/pull/724, but I have received no response from the upstream maintainers for more than 3 weeks. Would it be possible to include the patch files here in an effort to just get this port added?

**Comment by LilyWangLL** [comment]
Hi @Ace314159, thanks for your PR, we need wait for simbody/simbody#724 merged.

**Comment by Ace314159** [comment]
@LilyWangLL The upstream PR has been merged and I've made the appropriate changes to use it. Is there anything else you'd like me to do?

File: ports/simbody/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,38 @@
+vcpkg_fail_port_install(ON_TARGET "uwp")
```
Comment (LilyWangLL) [inline ports/simbody/portfile.cmake:1]:
```suggestion
```
`vcpkg_fail_port_install` has been deprecated, VCPKG use `supports` in vcpkg.json instead of it.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Review by LilyWangLL — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
LGTM! Thanks @Ace314159 :)

**Pull request merged**